etc/deploy_milvus_on_kubesphere.md

```diff
@@ -0,0 +1,89 @@
+# Deploy Milvus on KubeSphere
+
+Milvus is an open-source vector database for unstructured data.
+
+This tutorial demonstrates how to deploy Milvus on KubeSphere.
+
+## Prerequisites
+
+- To install KubeSphere 3.2.1 on Kubernetes, your Kubernetes version must be v1.19.x, v1.20.x, v1.21.x or v1.22.x (experimental)
+- Make sure your machine meets the minimal hardware requirement: CPU > 4 Core, Memory > 16 GB.
```

Comment:
The minimum hardware spec for the Milvus cluster is 16c 64g https://milvus.io/docs/v2.0.x/prerequisite-docker.md